roles/openshift_examples/tasks/main.yml

```diff
@@ -1,8 +1,45 @@
 ---
-- name: Copy openshift examples
-  copy:
-    src: "examples/{{ content_version }}/"
+######################################################################
+# Copying Examples
+#
+# We used to use the copy module to transfer the openshift examples to
+# the remote. Then it started taking more than a minute to transfer
+# the files. As noted in the module:
+#
+#   "The 'copy' module recursively copy facility does not scale to
+#   lots (>hundreds) of files."
+#
+# The `synchronize` module is suggested as an alternative, we can't
+# use it either due to changes introduced in Ansible 2.x.
+- name: Create local temp dir for OpenShift examples copy
+  local_action: command mktemp -d /tmp/openshift-ansible-XXXXXXX
+  become: False
+  register: g_copy_examples_mktemp
+  changed_when: False
+  run_once: True
+
+- name: Create tar of OpenShift examples
+  local_action: command tar -C "roles/openshift_examples/files/examples/{{ content_version }}/" -cvf "{{ g_copy_examples_mktemp.stdout }}/openshift-examples.tar" .
```

Comment:
So then the original command:

    +  local_action: command tar -C "roles/openshift_examples/files/examples/{{ content_version }}/" -cvf "{{ g_copy_examples_mktemp.stdout }}/openshift-examples.tar" .


Would turn into:

    +  local_action: command tar -C "{{ role_path }}/files/examples/{{ content_version }}/" -cvf "{{ g_copy_examples_mktemp.stdout }}/openshift-examples.tar" .

? Sounds reasonable to me.